python/src/scipp/plotting/__init__.py

```diff
@@ -142,6 +142,11 @@ def plot(*args, **kwargs):
         `"log"`. Defaults to `"linear"`.
     :type norm: str, optional
 
+    :param resampling_mode: Resampling mode. Possible choices are `"sum"` and
```

Comment:
Maybe for later. It might be good to exercise this via examples in the visualisation docs.